aquasec/resource_host_assurance_policy.go

```diff
@@ -660,7 +656,7 @@ func resourceHostAssurancePolicyUpdate(d *schema.ResourceData, m interface{}) er
 		if err == nil {
 			err1 := resourceHostAssurancePolicyRead(d, m)
 			if err1 == nil {
-				d.SetId(name)
+				d.SetId(iap.Name)
```

Comment:
duplicate it set, also in line 638

Reply:
638 is part of the create function this is the update function?